frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts

```diff
@@ -722,7 +722,7 @@ function _getOriginalTransformsForLayerOrNull(
   } else if (type === "thin_plate_spline") {
     const { source, target } = transformation.correspondences;
 
-    return createThinPlateSplineTransform(target, source, dataset.dataSource.scale.factor);
+    return createThinPlateSplineTransform(source, target, dataset.dataSource.scale.factor);
```

Comment:
this was an error before. since we don't have any tps in production, there shouldnt be any problems.